mapie/subsample.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+from typing import Any, Generator, List, Optional, Tuple
+
+import numpy as np
+from sklearn.model_selection import BaseCrossValidator
+from sklearn.utils import resample
+from sklearn.utils.validation import _num_samples
+
+from ._typing import ArrayLike
+
+
+class Subsample(BaseCrossValidator):  # type: ignore
+    """
+    Generate a sampling method, that resamples the training set with
+    possible bootstrap. It can replace KFold as cv argument in the MAPIE
```

Comment:
bootstraps ?

Reply:
It can replace ``KFold`` or ``LeaveOneOut``

Reply:
Fait